src/DIRAC/DataManagementSystem/Client/DataManager.py

```diff
@@ -53,30 +47,28 @@ def _isOlderThan(stringTime, days):
     return False
 
 
-def _initialiseAccountingObject(operation, se, files):
-    """create accouting record"""
-    accountingDict = {}
-    accountingDict["OperationType"] = operation
+def _initialiseMonitoringData(operation, se, files):
+    """create monitoring record"""
+    data = {}
```

Comment:
please do not use so generic words as `data` ( in particular in a class taking care of `data` meaning actual files :D) or `baseDict` 

I know it is not accounting anymore, but I'd rather you to keep `accountingDict` and the like, so we have a hint about what is contained in the dict